WooCommerce/Classes/Model/OrderPaymentMethod.swift

```diff
@@ -0,0 +1,33 @@
+/// Order Payment methods
+enum OrderPaymentMethod: RawRepresentable {
+    /// Cash on Delivery
+    case cod
```

Comment:
Please add something like `none` - in the REST API response it is an empty string. This is the payment method automatically assigned to Manually Created Orders and I think it is likely those orders will be something merchants use with Card Present payments (i.e. someone comes in the shop or calls on the phone, they enter the order in front of them and then the customer pays in person with their card when they get the goods.)